test/integration/lnd-utils.spec.ts

```diff
@@ -11,38 +12,77 @@ import {
   pay,
   waitFor,
 } from "test/helpers"
+import moment from "moment"
```

Comment:
If we're adding the `moment` package, we can also use it in other places across the codebase (as part of another PR) where we're relying on less intuitive date manipulation